rotkehlchen/chain/ethereum/modules/gearbox/decoder.py

```diff
@@ -0,0 +1,22 @@
+from typing import TYPE_CHECKING
+from rotkehlchen.chain.evm.decoding.gearbox.decoder import GearboxCommonDecoder
+
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.user_messages import MessagesAggregator
+
+
+class GearboxDecoder(GearboxCommonDecoder):
+    def __init__(
+            self,
+            ethereum_inquirer: 'EthereumInquirer',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: 'MessagesAggregator',
+    ) -> None:
+        super().__init__(
+            evm_inquirer=ethereum_inquirer,
+            base_tools=base_tools,
+            msg_aggregator=msg_aggregator,
+        )
```

Comment:
we were commenting during the weekend and this is not needed

```suggestion
class GearboxDecoder(GearboxCommonDecoder):
    ...
        
```